packages/@core.keystone/plugins/utils/index.js

```diff
@@ -1,12 +1,13 @@
 const { getType } = require('@keystonejs/utils')
 const { get } = require('lodash')
 
-const composeHook = (originalHook, newHook) => async params => {
+const composeHook = (originalHook, newHook, isResolvers = true) => async params => {
```

Comment:
please, add some description to new flag

Reply:
Please, it’s better not to add a new flag. Try to improve an existing logic.

Reply:
How should I improve existing logic, if you wrote util wrong?) Since validate hooks don't return anything...and you pass it new hook 😄

Reply:
Added schemaDoc

Reply:
I suggest two options:
1. Write new function which passes original `resolvedData` to both hooks
2. Rename `isResolvers` to `chainResolvedData = true` to make parameter purpose more understandable